### ISSUE TITLE
Issue with initializing session

### ISSUE THREAD

**Issue by laurinis** [rOpenGov/rwfs#13]
Hello,

I am trying to reproduce the Lighting data demo (http://rpubs.com/jlehtoma/210761)

but command `init_session(API_KEY)` returns,

Error: could not find function "init_session"

(I am running this on R Studio 1.0.136 , OS X 10.12.2 and R 3.3.2.)

I had some problems with ogr2ogr and gdal but manage to solve those issues thank to the help from previous issues described here :

https://github.com/OSGeo/proj.4/issues/351
https://github.com/rOpenGov/fmi/issues/6

**Comment by jlehtoma** [comment]
This is actually an issue with [`fmi`](https://github.com/rOpenGov/fmi) package, not `rwfs`. The division of functionality between the packages *is* confusing, so I can certainly understand the confusion.

Previously `init_session()` was still not merged to the `master` branch and hence not available unless `fmi` was installed from a different branch. However, this has now been fixed in `fmi` release 0.2.0. Try installing `fmi` again by:

```
devtools::install_github("ropengov/fmi")
```

Let me know if the lightning data demo doesn't work for you for some reason.

Note also, that on Mac you can avoid using ``ogr2ogr` completely by installing the development version of `rwfs`:

```
devtools::install_github("ropengov/fmi", ref = "develop")
```

**Comment by jlehtoma** [comment]
No recent updates here so I'm closing the issue for now.
